parcels/plotting.py

```diff
@@ -186,10 +193,14 @@ def plotfield(field, show_time=None, domain=None, projection=None, land=True,
 
     timestr = parsetimestr(field[0].grid.time_origin, show_time)
     titlestr = kwargs.pop('titlestr', '')
+    if fld.grid.zdim > 1:
+        depthstr = ' at depth %f ' % fld.grid.depth[depth_level]
```

Comment:
This works for z-levels.
For s-levels, maybe you could put a warning or something that says that you show a constant level, which does not correspond to a constant depth.